site/src/pages/CreateTemplateGalleryPage/StarterTemplates.tsx

```diff
@@ -23,14 +23,13 @@ const selectTags = (starterTemplatesByTag: StarterTemplatesByTag) => {
 };
 
 const sortVisibleTemplates = (templates: TemplateExample[]) => {
-	// The docker template should be the first template in the list,
-	// as it's the easiest way to get started with Coder.
-	const dockerTemplateId = "docker";
```

Comment:
Previously, the docker template was being moved to the front of the list in the frontend. We simply swapped that for the tasks-docker template.

Reply:
The docker template is just no longer the very first template displayed. It's still in the list we're sorting it's just ordered based on its name rather than being manually pulled to the front.

Reply:
Product wise I think these are meant to be different lists. The EmptyTemplates component is only ever seen when your deployment has *no* templates pushed, whereas the starter-templates page is more likely to be encountered by a typical developer looking to make a quick template.
But also I'm okay with simply making the docker template second here too.